modules/hyprland/hm.nix

```
@@ -19,11 +20,17 @@ let
     };
     misc.background_color = rgb base00;
   };
-
 in {
   options.stylix.targets.hyprland.enable =
     config.lib.stylix.mkEnableTarget "Hyprland" true;
 
-  config.wayland.windowManager.hyprland.settings =
-    lib.mkIf config.stylix.targets.hyprland.enable settings;
+  config = lib.mkIf config.stylix.targets.hyprland.enable {
+    wayland.windowManager.hyprland.settings = settings;
+
+    services.hyprpaper = {
+      enable = true;
+      settings.prelod = ["${config.stylix.image}"];
```

This appears to be a typo.

```suggestion
      settings.preload = ["${config.stylix.image}"];
```